advanced/Scripts/piholeDebug.sh

```diff
@@ -1027,7 +1027,7 @@ list_files_in_dir() {
                     # Check if the file we want to view has a limit (because sometimes we just need a little bit of info from the file, not the entire thing)
                     case "${dir_to_parse}/${each_file}" in
                         # If it's Web server error log, just give the first 25 lines
```

Comment:
Make sure you update the comment as well.

Reply:
Fixed.